apps/rush/src/utilities/PackageReviewChecker.ts

```diff
@@ -41,9 +39,18 @@ export default class PackageReviewChecker {
         const scope: string = Utilities.parseScopedPackageName(packageName).scope;
 
         // Make sure the scope isn't something like "@types" which should be ignored
-        if (!this._packageReviewConfiguration.ignoredNpmScopes.has(scope)) {
+        if (!this._rushConfiguration.approvedPackagesIgnoredNpmScopes.has(scope)) {
           // Yes, add it to the list if it's not already there
-          this._packageReviewConfiguration.addOrUpdatePackage(packageName, false, rushProject.reviewCategory);
+
+          // By default we put everything in the browser file.  But if it already appears in the
+          // non-browser file, then use that instead.
+          if (this._rushConfiguration.nonbrowserApprovedPackages.getItemByName(packageName)) {
+            this._rushConfiguration.nonbrowserApprovedPackages
+              .addOrUpdatePackage(packageName, rushProject.reviewCategory);
+          } else {
```

Comment:
nevermind

---
In reply to: [116122299](https://github.com/Microsoft/web-build-tools/pull/181#discussion_r116122299) [](ancestors = 116122299)